core/src/main/scala/org/apache/spark/storage/DiskBlockObjectWriter.scala

```diff
@@ -283,7 +283,7 @@ private[spark] class DiskBlockObjectWriter(
   }
 
   // For testing
-  private[spark] override def flush(): Unit = {
+  override def flush(): Unit = {
```

Comment:
[ERROR] [Error] /spark/core/src/main/scala/org/apache/spark/storage/DiskBlockObjectWriter.scala:286: weaker access privileges in overriding
def flush(): Unit (defined in class OutputStream)
  override should be public